packages/soundxyz/src/constants.ts

```diff
@@ -316,3 +316,7 @@ export const MINT_INFO_LIST_ABI = [
     type: 'function',
   },
 ]
+
+// for referrals
+export const ZORA_DEPLOYER_ADDRESS =
+  '0xe3bBA2A4F8E0F5C32EF5097F988a4d88075C8B48'
```

Comment:
What is the `ZORA_DEPLOYER_ADDRESS`?

Reply:
Ive been using it as the default referral address for Boost. It is used as a fallback here if no creatorAddress is provided.


It specifically an EOA that we currently use to collect referral income from Zora, Pods and Foundation.

Reply:
Will likely come back and update this once we get some clarity on where we want the referral income to go.